pubs/content.py

```diff
@@ -47,12 +48,11 @@ def system_path(path):
 
 
 def _open(path, mode):
-    if 'b' in mode:
+    if 'b' in mode or sys.version_info < (3,):
```

Comment:
I don't know if that would make it any clearer. If you have an idea, I am all hear.